is_core/utils/__init__.py

```diff
@@ -52,6 +54,34 @@ def flatten_fieldsets(fieldsets):
     return field_names
 
 
+def get_inline_views_from_fieldsets(fieldsets):
+    """
+    Returns a list of field names from an admin fieldsets structure.
+    """
+    inline_views = []
+    for _, opts in fieldsets or ():
+        if 'fieldsets' in opts:
+            inline_views += get_inline_views_from_fieldsets(opts.get('fieldsets'))
+        else:
+            if 'inline_view' in opts:
+                inline_views.append(opts.get('inline_view'))
+    return inline_views
+
+
+def get_inline_views_opts_from_fieldsets(fieldsets):
+    """
+    Returns a list of field names from an admin fieldsets structure.
+    """
+    inline_views = []
+    for _, opts in fieldsets or ():
+        if 'fieldsets' in opts:
```

Comment:
je to rekurzivní funkce, předávat to ještě přes jinou funkci kvli list comprehension to jenom zesložití